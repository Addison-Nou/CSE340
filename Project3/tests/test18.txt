{
    x1 : BOOLEAN;
    x1 = TRUE;
    x1 = ~ TRUE;
}