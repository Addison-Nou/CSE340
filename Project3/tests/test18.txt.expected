x1 3 2
x1 4 2